src/sdk/glacier_tapecloud.js

```diff
@@ -195,6 +195,8 @@ class TapeCloudUtils {
 }
 
 class TapeCloudGlacier extends Glacier {
+    static LOG_DELIM = ' -- ';
```

Comment:
Will we have an issue if we have this delimiter inside the entry itself?

Reply:
I think we won't have issue on the NooBaa side because the reader delimiter is still `\n` however `eeadm` doesn't really support this delimiter within the path either so I think there we would encounter some issues.